src/js/stores/schemes/healthChecksRowScheme.js

```diff
@@ -1,12 +1,15 @@
 var Util = require("../../helpers/Util");
 
 var HealthCheckProtocols = require("../../constants/HealthCheckProtocols");
+var HealthCheckPortTypes = require("../../constants/HealthCheckPortTypes");
 
 const healthChecksRowScheme = {
   protocol: HealthCheckProtocols.HTTP,
   command: null,
   path: null,
+  port: 0,
```

Comment:
This is necessary for the form field rows.
It's validated and post transformed correctly to the model.
```command``` and ```path``` co-exists too in this scheme.